docs/user-guide/multiple_sources.md

```diff
@@ -71,3 +71,5 @@ at that URL. If the `path` field is not set, Argo CD will use the repository sol
 !!! note
     Sources with the `ref` field set must not also specify the `chart` field. Argo CD does not currently support using  
     another Helm chart as a source for value files.
+
+The helm source that defines `ref` only needs `targetRevision` and `repoURL` fields. The other fields *(such as `path`)* are not supported.
```

Comment:
I think this is covered above:

> If the `path` field is set in the `$values` source, Argo CD will attempt to generate resources from the git repository at that URL. If the `path` field is not set, Argo CD will use the repository solely as a source of value files.

`path` actually is supported, it's just not required.